_targets.R

```diff
@@ -37,7 +36,11 @@ nhd_statsgo_parent_sbid <- '5728d6ace4b0b13d3918a992'
 ## 1_fetch/in. Clipped drb version already stored in 1_fetch/in on caldera. 
 Shangguan_dtb_cm_250m_clip_path <- '1_fetch/in/Shangguan_dtb_cm_250m_clip/w001001.adf'
 
+## vector of attributes to remove from the final static_inputs_nhm_combined dataframe because they are not ultimately used on any of the downstream model code 
+static_inputs_nhm_to_remove <- c('seg_width_empirical','reach_length_km','LENGTHKM_PRMS')
```

Comment:
If you agree with the comment I made in my review, I think we could just omit `seg_width_empirical` here:

```suggestion
static_inputs_nhm_to_remove <- c('reach_length_km','LENGTHKM_PRMS')
```